plutus-core/executables/uplc/Main.hs

```diff
@@ -268,64 +293,72 @@ runApplyToData (ApplyOptions inputfiles ifmt outp ofmt mode) =
                        Right (d :: Data) ->
                            pure $ UPLC.Program () ver $ mkConstant () d
 
+---------------- Benchmarking ----------------
+
+runBenchmark :: BenchmarkOptions -> IO ()
+runBenchmark (BenchmarkOptions inp ifmt semvar timeLim) = do
+  prog <- readProgram ifmt inp
+  let criterionConfig = defaultConfig {reportFile = Nothing, timeLimit = timeLim}
+      cekparams = mkMachineParameters semvar PLC.defaultCekCostModel
```

Comment:
Is it possible to use `PlutusCore.Evaluation.Machine.MachineParameters.Default.mkMachineParametersFor` here  instead? You'll need to apply it to `from PlutusCore.Evaluation.Machine.ExBudgetingDefaults.defaultCostModelParams` I think. Or something like that, the point is that we only track performance of builtins when `mkMachineParametersFor` is used and `mkMachineParameters` can be slower significantly (yes, even though the former uses the latter).